src/SimpleTopOpt.jl

```diff
@@ -1,6 +1,12 @@
 module SimpleTopOpt
-using TopOpt
 
-# TODO
+include("top88.jl")
```

Comment:
The file name has upper case `T`. Just rename the file.

Reply:
Ah interesting... had already made that change locally but git was set to ignore case. Should be fixed.